### PR TITLE
enhancement/Add constraints on columns in database tables #206

### PR DESCRIPTION
**Pull request by zetxdev**
Add constraints on columns in database tables #206

due to the fact that the most desired functionality to resolve this issue (addCheckConstraint) is included in paid licenses I proposed workaround with executing sql script from external file which is invoked from liquibase version management file anyway.

### CODE REVIEW

File: src/main/resources/db/changelog/constrains.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE financial_transactions ADD CHECK (amount > 0);
```
Comment (mirekgab) [inline src/main/resources/db/changelog/constrains.sql:1]:
```suggestion
ALTER TABLE financial_transactions ADD CONSTRAINT financial_transaction_check_amount CHECK (amount > 0);
```
Add constraint name.

**Review by mirekgab — CHANGES_REQUESTED** [review]

**Review by mirekgab — APPROVED** [review]

**Review by mirekgab — DISMISSED** [review]

File: src/main/resources/db/changelog/008-add-constraints-on-columns-in-database-tables.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: "10"
+      author: zetxdev
+      changes:
+        - sqlFile:
+            dbms: 'mysql'
+            path: db/changelog/constrains.sql
```
Comment (dawciobiel) [inline src/main/resources/db/changelog/008-add-constraints-on-columns-in-database-tables.yaml:8]: Please reneme name of thisfile to:
`008-add-constraints-on-columns-in-database-tables.sql`

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

**Review by dawciobiel — DISMISSED** [review]
Looks good to me

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Review by Iwona007 — APPROVED** [review]

**Pull request merged**